piecewise_web/js/ndt.js

```diff
@@ -8,20 +8,17 @@
 
 'use strict';
 
-var WebSocket;
-
-function NDTjs(server, serverPort, serverPath, callbacks, updateInterval, debugLevel) {
+function NDTjs(server, serverPort, serverProtocol, serverPath, callbacks,
+               updateInterval) {
   this.server = server;
   this.serverPort = serverPort || 3001;
   this.serverPath = serverPath || '/ndt_protocol';
+  this.serverProtocol = serverProtocol || 'ws';
```

Comment:
this is direct from the NDT repo, so I think we can leave it. I just wanted to gut check it, since this is for wss support.